src/Command/CommandSet.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Equip\Console\Command;
+
+use Equip\Structure\Set;
+
+class CommandSet extends Set
+{
+    
```

Comment:
You probably want a validation method in here to ensure the classes added extend the command interface.